src/components/base/RegisterForm.tsx

```diff
@@ -0,0 +1,118 @@
+import InputField from './Input'
+import Button from './Button'
+import { useState } from 'react'
+import { Credentials } from '../../entities/Credentials'
+
+export type InputFieldProps = {
```

Comment:
I would rename the prop to `RegisterFormProps`. Instead of listing all properties here again, I would have the Props be something like: 
```ts
export type RegisterFormProps {
    credentials: Credentials
}
```

Reply:
Renamed the prop to RegisterFormProps